tf2_geometry_msgs/package.xml

```diff
@@ -9,19 +9,22 @@
   <license>BSD</license>
 
   <url type="website">http://www.ros.org/wiki/tf2_ros</url>
-    
-  <buildtool_depend version_gte="0.5.68">catkin</buildtool_depend>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
 
   <build_depend>geometry_msgs</build_depend>
   <build_depend>tf2_ros</build_depend>
   <build_depend>tf2</build_depend>
   <build_depend>orocos_kdl</build_depend>
- 
-  <run_depend>geometry_msgs</run_depend>
-  <run_depend>tf2_ros</run_depend>
-  <run_depend>tf2</run_depend>
-  <run_depend>orocos_kdl</run_depend>
 
-  <test_depend>rostest</test_depend>
-</package>
+  <exec_depend>geometry_msgs</exec_depend>
+  <exec_depend>tf2_ros</exec_depend>
+  <exec_depend>tf2</exec_depend>
+  <exec_depend>orocos_kdl</exec_depend>
 
```

Comment:
you might have seen https://github.com/ros/geometry2/pull/217 by now. I opened this as a hack before I did that PR: I've incorporated it into this branch with https://github.com/ros2/geometry2/pull/7/commits/e924cb4e258b80ee80bbf85706c0f60e23c94e4f